routes/api.php

```diff
@@ -6,5 +6,6 @@
 
 Route::get('/quizzes', [QuizApiController::class, 'all']);
 Route::get('/quizzes/{id}', [QuizApiController::class, 'find']);
+Route::put('/quizzes/{quiz}', [QuizApiController::class, 'update']);
```

Comment:
As per my comment on the controller, because you're using route model binding here ({quiz}), laravel will automatically send a 404 response if the quiz doesn't exist, but the format of that response isn't what the front-end wants.

You can customise that response like this:

```
Route::put('/quizzes/{quiz}', [QuizApiController::class, 'update'])->missing(function () {
            return response()->json([
                'message' => 'Quiz not found',
            ], 404);
});
```